socialhome/streams/app/components/TagStampedElement.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+    <base-stamped-element :title="title" :help-text="helpText"/>
+</template>
+
+<script>
+import Vue from "vue"
+import "streams/app/components/BaseStampedElement.vue"
+
+
+export default Vue.component("tag-stamped-element", {
+    computed: {
+        title() {
+            return `#${this.$store.state.tagName}`
+        },
+        helpText() {
+            return interpolate(gettext("All content tagged %s"), [`#${this.$store.state.tagName}`])
```

Comment:
Used to be just the tag name, but if you want to put some more text that is fine. Make sure it ends in a period though ;) Also, I think it should be "All content tagged *with* %s".